### ISSUE TITLE
single-quoted strings represent charlists warning on Elixir 1.17.2 Erlang 27.0.1

### ISSUE THREAD

**Issue by samirbilalmemon** [pma/amqp#232]
On compilation with Elixir 1.17.2, Erlang 27.0.1 and the latest release of amqp on hex we get a warning for a line of code that has already been fixed here on the github repository.

Could you please do a new release of amqp implementing the changes on the github repo to deal with the below.

Warning:

```
==> amqp
Compiling 16 files (.ex)
     warning: single-quoted strings represent charlists. Use ~c"" if you indeed want a charlist or use "" instead
     │
 231 │       host: Keyword.get(options, :host, 'localhost') |> to_charlist(),
     │                                         ~
     │
     └─ lib/amqp/connection.ex:231:41

Generated amqp app
```

**Comment by ono** [comment]
v3.3.1 is out. Thanks!

**Comment by ono** [comment]
Reopening as 3.3.1 was a bad release...

**Comment by ono** [comment]
Released 3.3.2. Hopefully there will be no more compile issue 🙏
